README.md

```diff
@@ -15,70 +15,74 @@ Desktop App to interact with containers and pods running in Podman. Connect and
 - [**Code of Conduct**](#code-of-conduct)
 - [**License**](#license)
 
-## Join our Early Adopter Program!
+## Overview
 
-We are seeking users who are interested in helping us improve Podman Desktop.
 
-Register your interest to join our Early Adopter Program by filling <a href="https://forms.gle/ow73dV7Ce3YLzoXH7" target="_blank">this form</a>.
+Podman Desktop is a graphical interface that enables seamless development with containers and Kubernetes. 
```

Comment:
I think it's not accurate and precise enough.

Giving it a try:

Podman Desktop is a graphical interface that enables application developers to seamlessly work with containers and Kubernetes. 

Podman Desktop installs, configures and keeps Podman up to date on your local environment. It provides a system tray, to check status and interact with your container engine without losing focus from other tasks. The desktop application provides a dashboard to interact with containers, images, pods and volumes but also configure your environment with your OCI registries and network settings. Podman Desktop also provides capabilities to connect and deploy pods to Kubernetes environments. 

Podman Desktop also support multiple container engines, pick your favourite one and use the tool!